Assets/Editor/ModelInspector.cs

```diff
@@ -511,29 +618,36 @@ public override void OnPreviewGUI(Rect r, GUIStyle background)
                 return;
             }
 
+            if (m_Settings.displayMode == DisplayMode.Shaded)
+            {
+                EditorGUI.DrawRect(r, new Color( 0.1f, 0.1f, 0.1f, 1));
```

Comment:
This should probably be replaced with something defined in editor styles, but I wasn't sure what.

Reply:
I'm not convinced by that UI at all TBH. Who would ever find it intuitive (or useful)? It also does not look great at narrower inspector widths, e.g. 
![Clipboard01](https://user-images.githubusercontent.com/348087/75968532-3d23f380-5ed6-11ea-9726-e2f5bce9ae4e.png)

I'd rather add right-click-drag to PreviewRenderUtility; today it does nothing, and right-drag instead or orbiting the camera could orbit the light (in case of two lights, orbit both the same amount). That would also improve it for the material preview "for free".